ulib/LowStar.Literal.fsti

```diff
@@ -0,0 +1,137 @@
+module LowStar.Literal
+
+module B = LowStar.Buffer
+module MB = LowStar.Monotonic.Buffer
```

Comment:
ImmutableBuffer includes LowStar.Monotonic.Buffer. Since you are using only immutable buffers, you could avoid opening these modules, and use for example `IB.recallable`.